hibernate-vector/src/main/java/org/hibernate/vector/OracleByteVectorJdbcType.java

```diff
@@ -0,0 +1,182 @@
+package org.hibernate.vector;
+
+import java.sql.CallableStatement;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.BitSet;
+
+import org.hibernate.dialect.Dialect;
+import org.hibernate.sql.ast.spi.SqlAppender;
+import org.hibernate.type.SqlTypes;
+import org.hibernate.type.descriptor.ValueBinder;
+import org.hibernate.type.descriptor.ValueExtractor;
+import org.hibernate.type.descriptor.WrapperOptions;
+import org.hibernate.type.descriptor.java.JavaType;
+import org.hibernate.type.descriptor.jdbc.ArrayJdbcType;
+import org.hibernate.type.descriptor.jdbc.BasicBinder;
+import org.hibernate.type.descriptor.jdbc.BasicExtractor;
+import org.hibernate.type.descriptor.jdbc.JdbcLiteralFormatter;
+import org.hibernate.type.descriptor.jdbc.JdbcType;
+
+import oracle.jdbc.OracleConnection;
+import oracle.jdbc.OracleType;
+
+/**
+ * Specialized type mapping for single-byte integer vector {@link SqlTypes#VECTOR_INT8} SQL data type for Oracle.
+ *
+ * @author Hassan AL Meftah
+ */
+public class OracleByteVectorJdbcType extends ArrayJdbcType {
+
+
+	private static final byte[] EMPTY = new byte[0];
+
+	public OracleByteVectorJdbcType(JdbcType elementJdbcType) {
+		super( elementJdbcType );
+	}
+
+	@Override
+	public <T> JdbcLiteralFormatter<T> getJdbcLiteralFormatter(JavaType<T> javaTypeDescriptor) {
+		return null;
```

Comment:
Maybe just an FYI, but there is a literal format for VECTOR. The format is comma delimited numbers, within square brackets (same as what java.util.Arrays.toString(...) would return).

Example:
```sql
INSERT INTO example (vector_column) VALUES ('[0.1, 0.2, 0.3]')
```

Please note that I'm not familiar with Hibernate's code base; I'm not sure if JdbcLiteralFormatter is actually specified to generate a literal expression. The name implies it though, so I thought I should mention this.